pkg/config/application_ecs.go

```diff
@@ -47,6 +51,12 @@ type ECSDeploymentInput struct {
 	AutoRollback *bool `json:"autoRollback,omitempty" default:"true"`
 }
 
+func (in *ECSDeploymentInput) IsStandaloneTask() bool {
+	existClusterFile := in.ClusterDefinitionFile != ""
+	existVpcConfFile := in.VpcConfigurationFile != ""
+	return existClusterFile && existVpcConfFile
+}
+
```

Comment:
I think this check should depend on whether the input contains serviceDef or not.